modules/installation-bare-metal-agent-installer-config-yaml.adoc

```diff
@@ -54,7 +54,7 @@ Class E CIDR range is reserved for a future use. To use the Class E CIDR range,
 ====
 +
 <7> The subnet prefix length to assign to each individual node. For example, if `hostPrefix` is set to `23`, then each node is assigned a `/23` subnet out of the given `cidr`, which allows for 510 (2^(32 - 23) - 2) pod IP addresses. If you are required to provide access to nodes from an external network, configure load balancers and routers to manage the traffic.
-<8> The cluster network plugin to install. The supported values are `OVNKubernetes` (default value) and `OpenShiftSDN`.
```

Comment:
I believe that the original line should remain. both OVNKubernetes and OpenShiftSDN are supported for Compact and traditional clusters. It is only if you are building a SNO cluster, that the only option is OVNKubernetes.

Reply:
Addressed